### ISSUE TITLE
Add focus indicator (outline) on vf-tabs to assist keyboard navigation

### ISSUE THREAD

**Issue by nikiforosk** [visual-framework/vf-core#1758]
The [vf-tabs component](https://stable.visual-framework.dev/components/vf-tabs/) does not have visual a focus indicator to help people who navigate with the keyboard.

As mentioned in [the keyboard testing guidelines of Webaim](https://webaim.org/techniques/keyboard/#problems):

> A keyboard user typically uses the Tab key to navigate through interactive elements on a web page—links, buttons, fields for inputting text, etc. When an item is tabbed to, it has keyboard "focus" and can be activated or manipulated with the keyboard. A sighted keyboard user must be provided with a visual indicator of the element that currently has keyboard focus. Focus indicators are provided automatically by web browsers. While their appearance varies based on the browser, the focus indicator is typically shown as a border or highlight (called an outline) around the focused element.

[The vf-navigation component](https://stable.visual-framework.dev/components/vf-navigation/) shows an outline for the active tab:
![vf-navigation-outline-active-tab-Screenshot 2022-02-14 at 11 24 06](https://user-images.githubusercontent.com/20280139/153856762-6d9602d4-3d2a-47de-90da-70e86d493e7f.png)

However, when the keyboard focus moves to the active tab of vf-tabs (labelled "Section" below) the outline isn't shown:
![vf-tab-no-outline-active-tab-Screenshot 2022-02-14 at 11 24 20](https://user-images.githubusercontent.com/20280139/153856866-b08b32dc-278b-4eb7-a6eb-49926ad31b7a.png)

We should add the outline to vf-tabs too to assist keyboard navigation.

We came across this issue while we were testing keyboard navigation on the front page of the EBI training online tutorial: https://www.ebi.ac.uk/training/online/courses/bioinformatics-terrified/

**Comment by nikiforosk** [comment]
Also see related issue #1782

**Comment by sandykadam** [comment]
@pacope92 add ARIA tags to the tab component take example from vf-navigation component https://stable.visual-framework.dev/assets/vf-navigation/vf-navigation.js

**Comment by nikiforosk** [comment]
@kasprzyk-sz I see the outline in latest: https://latest.visual-framework.dev/components/vf-tabs/

There is a small change in tab behaviour, which we can talk about in the stand-up.

**Comment by nikiforosk** [comment]
@kasprzyk-sz I had a look at [the ARIA documentation of tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/#keyboard-interaction-21), and I think that we should be using left/right arrows to navigate from one tab to the next in a tab group. 

The tab key should move the focus to the next interactive element (link) within the selected tab (or if there aren't any links in the selected tab, it should move the focus to the next interactive element on the page), which is the current [tab behaviour in stable](https://stable.visual-framework.dev/components/vf-tabs/#vf-tabs__section--1).

**Comment by nikiforosk** [comment]
I can see the changes in the tab behaviour in latest, thank you @kasprzyk-sz!

**Comment by nikiforosk** [comment]
The changes have now been released: https://stable.visual-framework.dev/components/vf-tabs/
